com.chartboost.mediation/Runtime/AdFormats/Banner/Unity/ChartboostMediationUnityBannerAd.cs

```diff
@@ -250,14 +283,33 @@ private IChartboostMediationBannerView BannerView
                     _bannerView.DidRecordImpression += OnRecordImpression;
                     _bannerView.DidDrag += OnDrag;
                     
-                    _bannerView.SetVisibility(gameObject.activeSelf);
                     _bannerView.SetDraggability(Draggable);
                 }
         
                 return _bannerView;
             }
         }
 
+        private async void AutoLoadOnInit()
```

Comment:
Is this a method that publishers will use or is it internal/demo? This may cause banners to automatically be requested when our SDK initializes. We may not want to do that for them.

Reply:
Yeah I should remove it. It was something I initially added for my testing/debugging but ended up just keeping it. It is not part of TDD so I don't think It should be included